### PR TITLE
Add Readme of English version

### PR DESCRIPTION
**Pull request by TheBeatles1994**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,82 +3,81 @@
...
 
-通过合理的**容量规划**，用户可减少人力交付成本和运维成本，并可提高集群资源整体利用率。
+- **Capacity Planning**: according to the existing server specifications (including the number of CPU cores, size of memory, capacity of disk, etc) and application workloads files (including the replicas, affinity rules, resource requirements, etc), calculate the minimum number of nodes required to install the cluster and its applications successfully.
+- **Simulating Deploying Applications**: in the running kubernetes cluster, determine whether the applications can be deployed successfully at one time by simulating deploying applications in the fake mirror cluster. If the cluster size does not meet the resource requirements of applications, a minimum cluster expansion proposal is given to solve the All-or-Nothing problem of deploying applications;
+- **Pods Migration**: in the running Kubernetes cluster, pods can be migrated between nodes according to the migration policy(such as cluster shrink, defragmentation, etc).
```
Comment (Liwling) [inline README.md:18]: shrink 应改为 downsize

File: README.md [hunk 2]
```diff
@@ -3,82 +3,81 @@
...
-- [x] 支持模拟 Kube-Scheduler 调度并给出应用部署拓扑结果
-- [x] 支持自动添加节点以满足应用成功部署
-- [x] 支持模拟 [Open-Local](https://github.com/alibaba/open-local) 存储调度
-- [x] 支持解析 Helm Chart
-- [x] 支持设置集群资源水位
-- [x] 支持设置 Workload 部署顺序
-- [ ] 支持解析 CR 资源
-- [ ] 支持处理 PV/PVC 资源
-- [ ] 支持清理空闲节点
+- [x] Support to simulate Kube-Scheduler and reports the topology results of applications deployment
```
Comment (Liwling) [inline README.md:31]: reports -> report

File: README.md [hunk 3]
```diff
@@ -92,14 +91,14 @@ spec:
     - name: more_pods
       path: example/application/more_pods
 
-  # newNode: 导入调整集群规模的节点配置文件，节点规格可根据需求任意指定。目前只支持配置一个节点
+  # the specification of added node, which can be file path or folder path
```
Comment (Liwling) [inline README.md:94]: 需要添加的，added表示已添加，改为node to be added

File: README.md (hunk 1 above)
Comment (TheBeatles1994) [reply README.md]: scaling down cluster 更好一些：https://docs.oracle.com/en/operating-systems/olcne/1.1/start/scale-down.html

File: README.md (hunk 2 above)
Comment (TheBeatles1994) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (TheBeatles1994) [reply README.md]: done

**Pull request merged**